File: 2_method.py
class Car:
    def __init__(self, brand):
        self.brand = brand
        print(f"The constructor is called for {self.brand}")

    def engine_start(self):
        print(f"Starting the engine of brand : {self.brand}")

    def drive(self, speed):
        print(f"{self.brand} is driving at speed {speed} km/h")

    @staticmethod
    def alert():
        print("peep peep!")

my_car = Car("Toyota")

my_car.engine_start()
my_car.drive(60)

Car.alert()

File: 2_method.cpp
#include <bits/stdc++.h>
using namespace std;

class Car {
public:
    string brand;

    Car(string b) {
        brand = b;
        cout << "The constructor is called for " << brand << endl;
    }

    void engine_start() {
        cout << "Starting the engine of brand : " << brand << endl;
    }

    void drive(int speed);

    static void alert() {
        cout << "peep peep !" << endl;
    }
};

void Car::drive(int speed) {
    cout <<"Car : " << brand << " is driving at speed of" << speed << " km/h" << endl;
}

int main() {
    Car myCar("Mayback");

    myCar.engine_start();
    myCar.drive(60);

    Car::alert();

    return 0;
}
